### ISSUE TITLE
Voicepack is not usable if the are brackets behind a specific country.

### ISSUE THREAD

**Issue by sandpatch** [skiselkov/BetterPushbackC#329]
A package called "en_IT(xxx)" will not work. It seems to only work if the pack is just the country code and nothing else. E.g. "pl(xxx)" will work but not "en-pl(xxx)" or "en_PL(xxx)".

**Comment by pjoerg** [comment]
After a lot of trial and error, I have found that the format "en_XY(xxx)" actually does work, if there is a "en_XY" (i.e. without suffix) folder present. I have changed all my language files to this format, using single digit suffixes. and BPB chooses randomly between the localized subsets, including the unsuffixed local base set. Please see the screenshot of my "msgs"-folder.

![grafik](https://user-images.githubusercontent.com/75171420/100518016-41dcb600-318f-11eb-8f60-9a5c77b452d7.png)

**Comment by sandpatch** [comment]
@pjoerg 
Wow thanks. I tired it as well and it works! This is awesome, can't imagine how much you tried. I will see if I can implement it.
Unfortunately I can't get it to work for Norway though

**Comment by pjoerg** [comment]
You're right, Norway somehow doesn't seem to work. What also doesn't work for me - though I don't know if it was ever supposed to - is switching to the new local voice when doing a turnaround. I always end up with the voice from my first departure airport. It changes correctly, though, when I dis- then reenable BPB in the plugin admin.
 
It would be great, if BPB could somehow read out the current airport location at each start of the routine, not only at the start of the flight...

**Comment by sandpatch** [comment]
Yeah, I think that merits it's own thread. I know you can bypass by opening the settings and saving. The first time it takes time (like a minute) but it doesn't crash. After that it's instantaneous and re selects the local pack
